src/main/resources/instructorEmailFragment-instructorCopyPreamble.html

```diff
@@ -1,5 +1,13 @@
 <p>
   The email below has been sent to students of course: [${courseId}] ${courseName}.
   <br><br>
+  Kindly note that this email simply serves as a preview of how the email will appear to the
+  students, and the link is not the actual link that the students will receive.
+  As such, please do not forward this email to students.
+  We recommend that you make the following announcement (edit content as you see fit) using an alternative means (e.g., your course announcements) to alert students:
+  <hr>
+  The TEAMMATES session ... in course ... is now open.
```

Comment:
Since we already have access to `courseId` and `courseName` here, we can fill it in instead of having ... as placeholders. 

Maybe we can take a look and see if we can include the feedback session name as well.

Reply:
Added both courseId and courseName.